release-team/role-handbooks/branch-manager/README.md

```diff
@@ -1,235 +1,307 @@
+1. Select shadows and guide them in preparation to become the section lead for the next cycle.
+1. Update this handbook where appropriate for the next release cycle.
+1. Have willingness to accommodate with different timezone esp. for the release team and your shadows.
+1. Participate in conversations that happen on [#sig-release] and [#release-managers]
+
+To get a better overview of the time it takes to run the release tools, you can take a look at the collected metrics from each of these [release cut issues][release-cut-issues] on the sig-release repo.
+
+[Branch Management diagram](https://docs.google.com/presentation/d/1usEbwHMSC8vR7HvbxHJBOj2ISdTkw9rmufQUq7fkIl4/edit?ts=5cc0de62#slide=id.g577153f745_1_405) illustrating the process over a 3 month [release lifecycle](https://docs.google.com/presentation/d/1usEbwHMSC8vR7HvbxHJBOj2ISdTkw9rmufQUq7fkIl4/edit?ts=5cc0de62#slide=id.g577153f745_3_0).
+
+[Kubernetes]: https://github.com/kubernetes/kubernetes/releases
+[release-1.15]: https://github.com/kubernetes/sig-release/tree/master/releases/release-1.15
+[#sig-release]: https://kubernetes.slack.com/messages/C2C40FMNF
+[#release-mangers]: https://kubernetes.slack.com/messages/CJH2GBF7Y
+[release-cut-issues]: https://github.com/kubernetes/sig-release/issues?utf8=%E2%9C%93&q=is%3Aissue+1.15.0+is%3Aclosed+
 
 ## Minimum Skills
 * Familiarity with basic Unix commands and able to debug shell scripts.
 * General knowledge of Google Cloud (Cloud Build and Cloud Storage).
 * Open to seeking help and can communicating clearly.
 
 # Prerequisite
-* Contact [Kubernetes Build Admins][kubernetes-build-admins], identifying yourself as the incoming release branch manager for the current release team and requesting to be added to the special privilege group for the GCP project used to build the releases
-   * TODO: should the release lead also request some type of read access here for themselves?
-   * TODO: should the branch manager shadows also get at least read access at the beginning?  If a shadow is going to exercise the process on behalf of the lead, they will need full read/write access ahead of time.
-   * TODO: can the group name be identified here in a clear/transparent way?
-* Ask the list owner(s) to give you access to **post** to these mailing lists:
-   * [kubernetes-announce][k-announce-list] via owner contact form [here][k-announce-request]
+A list of To Do(s) to get started as Branch Manager:
+
+Important:
+> - [ ] Request to be a [Kubernetes community org member](https://github.com/kubernetes/community/blob/master/community-membership.md#member).
+
+- [ ] Contact [Kubernetes Build Admins][kubernetes-build-admins], to identify yourself as the incoming release branch manager for the current release team and request to be added to the special privilege group for the `kubernetes-release-test` GCP project that's used to build the releases
+   * **NB**: should the release lead also request some type of read access here for themselves?
```

Comment:
Rather than changing "TODO" to "NB" let's just open an issue for each of these lines and delete the lines.    Instead add a line "Similar access should later in the cycle be granted to shadows depending on how they progress during the cycle and what actions they prove able to step up to exercise."

Reply:
The TODO/NB item is noted in #696 and the add _Similar access should later..._ is added to the handbook.